webui/react/src/components/Searches/Searches.tsx

```diff
@@ -873,7 +875,16 @@ const Searches: React.FC<Props> = ({ project }) => {
       <div className={css.content} ref={contentRef}>
         {!isLoading && experiments.length === 0 ? (
           numFilters === 0 ? (
-            <NoExperiments />
+            <Message
```

Comment:
qq: what is the reason to replace `<NoExperiments />`?

Reply:
Want it to say "Searches" instead of "Experiments"